.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
         language_version: python
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.812
+    rev: v0.971
```

Comment:
This fixes a compilation error when `pre-commit` attempts to install `mypy` on my M2 MacBook Pro. The `pre-commit` author on Stack Overflow [suggests](https://stackoverflow.com/a/71298400) fixing the issue by updating to a recent version of mypy.